src/main/java/ch/njol/skript/expressions/ExprPlayerViewDistance.java

```diff
@@ -25,42 +25,36 @@
 import ch.njol.skript.doc.Name;
 import ch.njol.skript.doc.RequiredPlugins;
 import ch.njol.skript.doc.Since;
-import ch.njol.skript.expressions.base.PropertyExpression;
-import ch.njol.skript.lang.Expression;
-import ch.njol.skript.lang.SkriptParser;
-import ch.njol.util.Kleenean;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
 import ch.njol.util.coll.CollectionUtils;
+import org.apache.commons.lang.NotImplementedException;
 import org.bukkit.Bukkit;
 import org.bukkit.entity.Player;
 import org.bukkit.event.Event;
 import org.eclipse.jdt.annotation.Nullable;
 
 @Name("View Distance")
-@Description("The view distance of a player. Can be changed.")
-@Examples({"set view distance of player to 10", "set {_view} to view distance of player",
-		"reset view distance of all players", "add 2 to view distance of player"})
-@RequiredPlugins("Paper 1.9-1.13.2")
+@Description({
+	"The view distance of a player. Can be changed.",
+	"NOTE: This may not work on some versions (such as MC 1.14.x).",
+	"The return value in this case will be the view distance set in system.properties."})
+@Examples({
+	"set view distance of player to 10", "set {_view} to view distance of player",
+	"reset view distance of all players", "add 2 to view distance of player"})
```

Comment:
```suggestion
	"reset view distance of all players", "add 2 to view distance of player"
})
```
Just for formatting :)